x/twap/keeper_test.go

```diff
@@ -14,9 +14,10 @@ import (
 
 // TODO: Consider switching this everywhere
 var (
-	denom0            = "token/B"
-	denom1            = "token/A"
-	defaultUniV2Coins = sdk.NewCoins(sdk.NewInt64Coin(denom0, 1_000_000_000), sdk.NewInt64Coin(denom1, 1_000_000_000))
+	denom0            = "token/A"
+	denom1            = "token/B"
```

Comment:
why are we switching / changing denom order?

Reply:
Actually I think this is fine

Reply:
Yes, I did not change the order in tests. Only renamed here so that I can add another `denom2` == `token/C` while having the identifier ordering to still make sense